src/main/java/de/hbz/lobid/helper/EtikettMaker.java

```diff
@@ -166,6 +166,7 @@ private void initMaps(InputStream labelIn) {
 		try (InputStream in = labelIn) {
 			return new ObjectMapper().readValue(in, type);
 		} catch (Exception e) {
+			e.printStackTrace();
```

Comment:
hm, in theory this should not be necessary. Following the call stack, the stack trace will be  printed at: https://github.com/hbz/lobid-rdf-to-json/blob/master/src/main/java/de/hbz/lobid/helper/EtikettMaker.java#L121 , maybe logging level was set to something >debug ?